debian/rules

```diff
@@ -25,6 +26,7 @@ configure_options = \
 	--with-grub2-mkconfig-path=/usr/sbin/grub-mkconfig \
 	--with-systemdsystemgeneratordir=/lib/systemd/system-generators \
 	--with-systemdsystemunitdir=/lib/systemd/system \
+	BASH_COMPLETIONSDIR=/usr/share/bash-completion/completions \
```

Comment:
Any reason not to rely on pkg-config here?

Reply:
IIRC, for the same reason we don’t rely on it for the `--with-systemd*` arguments — it can end up retrieving a path in the wrong prefix.